build/decisions/bdr014-menstrual-leave-policy.html

```diff
@@ -78,17 +78,18 @@
       <div class="bordered-section">
 	<h2 class="section-heading">Menstrual Leave Policy</h2>
 	<h3 class="section-heading"><u>context </u></h3>
-	<p class="section-content">Many countries and companies around the globe already offer some form of leave which coincides with menstrual discomfort. The existence of such a policy only seems logical and civilized. The question therefore is one of presentation: how can we avoid engendering confusion/FUD? How can the policy be self-explanitory, to this end? Even among educated men & women there can be confusion about such a policy being “unfair” or surface confusion which conflates such a policy with sick leave</p>
+	<p class="section-content">Many countries and companies around the globe already offer some form of leave which coincides with menstrual discomfort. The existence of such a policy only seems logical and civilized. The question therefore is one of presentation: how can we avoid engendering confusion/FUD? How can the policy be self-explanatory, to this end? Even among educated men & women there can be confusion about such a policy being “unfair” or surface confusion which conflates such a policy with sick leave</p>
 
 	<h3 class="section-heading"><u>Decision </u></h3>
-	<p class="section-content">We have articled the geneis of our policy [link]. We have a no-questions-asked menstrual leave policy [link]. We will have ongoing discussion with the intention of destroying the taboos associated with menstruation.  </p>
+	<p class="section-content">We have articled the genesis of our policy [link]. We have a no-questions-asked menstrual leave policy [link]. We will have ongoing discussion with the intention of destroying the taboos associated with menstruation.  </p>
```

Comment:
@deobald Not related to this PR, but this link doesn't currently point to anything. What should it be pointing to?